src/tests/suite/daos_pool.c

```diff
@@ -1118,6 +1118,8 @@ label_strings_test(void **state)
 					     "0b101010",
 					     /* len=DAOS_PROP_LABEL_MAX_LEN */
 					     "0123456789ABCDEF0123456789ABCDEF0123456789ABCDEF0123456789ABCDEF0123456789ABCDEF0123456789ABCDEF0123456789ABCDEF0123456789ABCDE",
+					     "a-b-cde",
+					     "thiswoul-dntp-arse-asau-uidsoitsfine",
```

Comment:
maybe add a "valid" label that is 36 characters in length, has hyphens in all the right positions of the string, but contains at least one digit that would fail the isxdigit() test?
for example
"g006b637-c63a-4734-99bc-a71298597de1"

Reply:
Done.
```
[ RUN      ] POOL15: label property string validation
Verify 16 valid labels
mypool should be valid
my_pool should be valid
MyPool should be valid
MyPool_2 should be valid
cae61c0752f54874ad213c0ec43005cb should be valid
bash should be valid
Pool_ProjectA:Team42 should be valid
ProjectA.TeamOne should be valid
server42.fictionaldomaincae61c07.org should be valid
0ABC should be valid
0xDA0S1234 should be valid
0b101010 should be valid
0123456789ABCDEF0123456789ABCDEF0123456789ABCDEF0123456789ABCDEF0123456789ABCDEF0123456789ABCDEF0123456789ABCDEF0123456789ABCDE should be valid
a-b-cde should be valid
thiswoul-dntp-arse-asau-uidsoitsfine should be valid
g006b637-c63a-4734-99bc-a71298597de1 should be valid
```